assignment/scripts/function-practice.js

```diff
@@ -1,79 +1,96 @@
-console.log('***** Function Practice *****')
+console.log("***** Function Practice *****");
 
 // Add the required code to complete the functions below
 // After _each_ function, use a console log to call the function
 // to test it and display the result
 
 // 1. Function to return 'Hello World!'
 function hello() {
-  return 'Hello World!';
+  return "Hello World!";
 }
 // Call the function to test
-console.log('Test - should say "Hello World!"', hello());
-
+console.log('Test - should say "Hello World!"', hello()); // moved this back to the original!
 
 // 2. Function to return an personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
-function helloName( name ) {
-  return;
-}
+function helloName(name) {
+  return console.log(`Hello, ${name}!`);
+} //
 // Remember to call the function to test
-
+helloName("David");
+helloName("Rebecca"); // Outputted both names correctly! Win! Stuck in term!
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
-  // return firstNumber + secondNumber;
+
+function addNumbers(num1, num2) {
+  let sum = num1 + num2;
+  return sum;
```

Comment:
Variables are great when you need to re-use a value or when it helps break up an otherwise long line of code to make it more readable. In this case, I think there is nothing wrong with just returning the sum directly.

```js
function addNumbers(num1, num2) {
  return num1 + num2;
}
```